src/ess/amor/load.py

```diff
@@ -154,15 +116,22 @@ def load_amor_detector_rotation(fp: Filename[RunType]) -> DetectorRotation[RunTy
     return sc.scalar(nu['value'].data['dim_1', 0]['time', 0].value, unit='deg')
 
 
+def load_amor_angle_from_horizon_to_center_of_incident_beam(
+    fp: Filename[RunType],
+) -> AngleCenterOfIncomingToHorizon[RunType]:
+    (kad,) = load_nx(fp, "NXentry/NXinstrument/master_parameters/kad")
+    return sc.scalar(kad['value'].data['dim_1', 0]['time', 0].value, unit='deg')
```

Comment:
It's stored as an `NXLog` and when we read that it ends up this way, but it doesn't make sense for this to be anything other than a scalar, so we only read the first value.

Reply:
Ok. Can you check that there is indeed a single entry? Otherwise, it may silently mess up when there is a time-dependent log. You could just use `squeeze`.

Reply:
In that case, please add a comment that explains the choice and why it is ok to use the first element.